package.json

```diff
@@ -22,6 +22,6 @@
     "pm2": "^4.2.3"
   },
   "dependencies": {
-    "pg": "^8.0.0"
+    "pg": "^8.0.2"
```

Comment:
mailgun should appear in dependencies, add  `$ npm i --save mailgun-js`

Reply:
Also you don't have the latest devDependencies and dependencies. You need to go to master and `$ git pull` then on your branch `$ git pull origin master`